packages/public/server/src/module/Api/src/ApiManager.php

```diff
@@ -76,6 +77,21 @@ public function getAliasData(AliasInterface $alias)
         ];
     }
 
+    public function getAliasDataFromUser(
```

Comment:
```suggestion
    public function getAliasDataForUser(
```
reads better IMHO.

Reply:
I'd even think about moving the instance part up since it doesn't affect the user (and you just polyfill it so that it works in every instance). So you could do that in the `ApiController` instead (that is aware of instances).